quarkus-petclinic/README.adoc

```diff
@@ -0,0 +1,51 @@
+= Quarkus Petclinic project
+
+== Run Locally
+
+=== Create css
+
+[source, bash]
+----
+./mvnw compile -Pcss
+----
+
+=== Dev mode
+
+[source, bash]
+----
+./mvnw compile quarkus:dev
+----
+
+=== Production jar
+
+Start docker container:
+
+[source, bash]
+----
+docker run --rm=true -e POSTGRES_USER=petclinic -e POSTGRES_PASSWORD=petclinic -e POSTGRES_DB=petclinic -p 5432:5432 postgres:14
+----
+
+Start the application:
+
+[source, bash]
+----
+java -Dquarkus.datasource.jdbc.url=jdbc:postgresql://localhost:5432/petclinic -Dquarkus.datasource.username=petclinic -Dquarkus.datasource.password=petclinic -jar target/quarkus-app/quarkus-run.jar
```

Comment:
When building this locally the runner jar has name "quarkus-petclinic-1.0.0-SNAPSHOT-runner.jar", minor nit but could be updated in the example command

Reply:
Ah, nice catch. I didn't even try these README directions, this was just a copy-paste from the original repo. But it's there and may as well be usable.